Source/WebInspectorUI/UserInterface/Models/RecordingAction.js

```diff
@@ -115,6 +115,39 @@ WI.RecordingAction = class RecordingAction extends WI.Object
         return typeof propertyDescriptor.value === "function";
     }
 
+    static bitfieldNamesForParameter(type, name, value, index, count)
+    {
+        if (value === 0)
+            return null;
+
+        function test_and_clear_bit(names, value, bit, desc) {
+            if (value & bit)
+                names.push(desc);
+            return value & ~bit;
+        }
+
+        let prototype = WI.RecordingAction._prototypeForType(type);
+        let names = [];
+
+        if ((name === "clear" && index === 0 && (type === WI.Recording.Type.CanvasWebGL || type === WI.Recording.Type.CanvasWebGL2)) ||
+            (name === "blitFramebuffer" && index === 8 && type === WI.Recording.Type.CanvasWebGL2)) {
+            value = test_and_clear_bit(names, value, prototype.COLOR_BUFFER_BIT, "COLOR_BUFFER_BIT");
```

Comment:
NIT: one thing we could do instead of repeating `COLOR_BUFFER_BIT` twice is to use the `description` to get the `bit`
```
function testAndClearBit(name) {
    let bit = prototype[name];
    if (!bit)
        return;

    if (value & bit)
        names.push(name);

    value = value & ~bit;
}
```

also, if you move the `let names = [];` above you don't need to pass it into every `testAndClearBit` call :)

you can already implicitly capture `value` because it's provided as a parameter to the parent function

Reply:
Thanks, that's a lot better than what I had.